src/backend/src/services/change-request.services.ts

```diff
@@ -0,0 +1,556 @@
+   * @param leftoverBudget  the leftover budget
+   * @param confirmDone  whether or not to confirm
+   * @returns the id of the created cr
+   * @throws if user is not allowed to create crs, if wbs element does not exist, or if the cr type is not stage gate
+   */
+  static async createStageGateChangeRequest(
+    submitter: User,
+    carNumber: number,
+    projectNumber: number,
+    workPackageNumber: number,
+    type: CR_Type,
+    leftoverBudget: number,
+    confirmDone: boolean
+  ): Promise<Number> {
+    // verify user is allowed to create stage gate change requests
+    if (submitter.role === Role.GUEST) throw new AccessDeniedException();
+
+    // verify wbs element exists
+    const wbsElement = await prisma.wBS_Element.findUnique({
+      where: {
+        wbsNumber: {
+          carNumber,
+          projectNumber,
+          workPackageNumber
+        }
+      }
+    });
+    if (!wbsElement) throw new NotFoundException('WBS Element', `${carNumber}.${projectNumber}.${workPackageNumber}`);
+
+    const createdChangeRequest = await prisma.change_Request.create({
+      data: {
+        submitter: { connect: { userId: submitter.userId } },
+        wbsElement: { connect: { wbsElementId: wbsElement.wbsElementId } },
+        type,
+        stageGateChangeRequest: {
+          create: {
+            leftoverBudget,
+            confirmDone
+          }
+        }
+      },
+      include: {
+        wbsElement: {
+          include: {
+            workPackage: {
+              include: {
+                project: { include: { team: { include: { leader: true } }, wbsElement: true } }
+              }
+            }
+          }
+        }
+      }
+    });
+
+    const team = createdChangeRequest.wbsElement.workPackage?.project.team;
+    if (team) {
+      const slackMsg =
+        `${submitter.firstName} ${submitter.lastName} wants to stage gate ${createdChangeRequest.wbsElement.name}` +
+        ` in ${createdChangeRequest.wbsElement.workPackage?.project.wbsElement.name}`;
+      await sendSlackChangeRequestNotification(team, slackMsg, createdChangeRequest.crId);
+    }
+
+    return createdChangeRequest.crId;
+  }
+
+  /**
+   * 
+   * @param submitter  The user creating the cr
+   * @param carNumber  the car number for the wbs element
+   * @param projectNumber  the project number for the wbs element
+   * @param workPackageNumber  the work package number for the wbs element
+   * @param type  the type of cr
+   * @param what  the description of the change
+   * @param why  the reason for the change
+   * @param budgetImpact  the impact on the budget
+   * @returns  the id of the created cr
+   * @throws if user is not allowed to create crs, if wbs element does not exist, or if the cr type is not standard
+   */
+  static async createStandardChangeRequest(
+    submitter: User,
+    carNumber: number,
+    projectNumber: number,
+    workPackageNumber: number,
+    type: CR_Type,
+    what: string,
+    why: any,
```

Comment:
this shouldn't be `any`, you can look at the validation on teh route to see what values it should have